test/useWizard.test.tsx

```diff
@@ -37,6 +38,20 @@ describe('useWizard', () => {
     expect(result.current.stepCount).toBe(2);
   });
 
+  test('should set step count to one when using falsy step', () => {
```

Comment:
@devrnt Had a quick review and spot two things:

1. Isn't the behavior tested twice now? An optional step is already added in `renderUseWizardHook()`. Hence, the test above (`'should set step count to two'`) should already capture this?
2. If you still prefer to have a separate (personally, I would just remove this test), more explicit test, then the name of the test doesn't reflect the implementation of the test.
2.1 It should either say `set step count to _two_` or one step should be removed.
2.2 Extreme nitpick, but "falsy" isn't entirely true. E.g. `""` and `0` would render. I would call it "non-rendered" or something.

Reply:
You're absolutely right, "falsy" is wrong here, don't know why I put it there :) 

Anyway I changed the test case, if you're interested you can [check it out](https://github.com/devrnt/react-use-wizard/commit/2129937a5f1a39f84b078460c56abd450bee8459)

Reply:
All right, that's better 👍

I'd still argue the optional step isn't needed in the default render hook test wrapper, but no biggie.